src/main/scala/io/citrine/lolo/linear/MultiTaskLinearRegression.scala

```diff
@@ -0,0 +1,49 @@
+package io.citrine.lolo.linear
+
+import io.citrine.lolo.api.{MultiTaskLearner, MultiTaskModel, MultiTaskTrainingResult, ParallelModels, TrainingRow}
+import io.citrine.random.Random
+
+case class MultiTaskLinearRegressionLearner(regParam: Option[Double] = None, fitIntercept: Boolean = true)
```

Comment:
I could definitely imagine cases in which someone would want to specify a single regParam per label. But our `Learner`s (and the associated settings structures in Mithril) are defined such that they do not need to know the # of labels when constructed. So that would require some slightly annoying validation on # params = # labels.

So I just left it at a single parameter for now.

Reply:
In the long run in Mithril it might make more sense for us to spin up a bunch of single-task linear learners instead of using multi-task linear learner, because then we can optimize the regularization parameter of each one individually. But this will work for now